app/index.js

```diff
@@ -719,6 +727,7 @@ JhipsterGenerator.prototype.app = function app() {
     if (this.databaseType == 'sql' || this.databaseType == 'mongodb') {
         this.template('src/main/java/package/security/_SpringSecurityAuditorAware.java', javaDir + 'security/SpringSecurityAuditorAware.java', this, {});
     }
+    this.template('src/main/java/package/security/_AppUser.java', javaDir + 'security/' + this.capitalizedBaseName + 'User.java', this, {});
```

Comment:
We could just name it "CustomUserDetails" that's what we have elsewhere, so we don't have the name clashes. Besides, all JHipster apps would have the same class, it might be easier to use.